doc/contributing/docs/api.rst

```diff
@@ -82,7 +88,7 @@ Configuration parameters
 For every configuration parameter, list the following details:
 
 *   Since which Tarantool version
```

Comment:
We need to change the guideline and example by explaining how to write the version info -- use Sphinx role or reference to the corresponding release note (RN) document, for example,
```
Since :doc:`2.10.0 </release/2.10.0>`.
```